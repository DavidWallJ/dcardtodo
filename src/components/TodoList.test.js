test('sample', () => {
	expect(true).toBe(false);
});
