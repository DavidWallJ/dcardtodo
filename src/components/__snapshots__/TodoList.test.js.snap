// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class TodoList extends _react.Component {constructor(...args) {var _temp;return _temp = super(...args), this.
		renderTodos = () => {
			const { todos } = this.props;
			return todos.map(todo => {
				return _react2.default.createElement(_TodoItem2.default, Object.assign({ key: todo.id }, todo, { __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: this }));
			});
		}, _temp;}
	render() {
		return (
			_react2.default.createElement('div', { __source: { fileName: _jsxFileName, lineNumber: 14 }, __self: this },
				this.renderTodos()));


	}} should exist 1`] = `
<div>
  <div>
    <p>
      <div
        className="checkbox"
        style={undefined}
      >
        <label
          title=""
        >
          <input
            checked={undefined}
            disabled={false}
            onChange={undefined}
            type="checkbox"
          />
          Show completed todos
        </label>
      </div>
      Walk the walk
    </p>
  </div>
  <div>
    <p>
      <div
        className="checkbox"
        style={undefined}
      >
        <label
          title=""
        >
          <input
            checked={undefined}
            disabled={false}
            onChange={undefined}
            type="checkbox"
          />
          Show completed todos
        </label>
      </div>
      Talk the talk
    </p>
  </div>
  <div>
    <p>
      <div
        className="checkbox"
        style={undefined}
      >
        <label
          title=""
        >
          <input
            checked={undefined}
            disabled={false}
            onChange={undefined}
            type="checkbox"
          />
          Show completed todos
        </label>
      </div>
      Walk the cat
    </p>
  </div>
</div>
`;
